frontend_react/src/shared/ui/components/custom/small-card.tsx

```diff
@@ -0,0 +1,77 @@
+import { useCallback } from 'react';
+import { Check, Edit, MoreHorizontal } from 'lucide-react';
+
+/**
+ * Props for the SmallCard component.
+ *
+ * @property {string} title - タスクのタイトル (The title of the task)
+ * @property {string} deadline - タスクの締切 (The deadline of the task)
+ */
+export type SmallCardProps = {
+  /* タスクのタイトル */
+  title: string;
+  /* タスクの締切 */
+  deadline: string;
+};
+
+export const SmallCard = ({ title, deadline }: SmallCardProps) => {
+  const handleEdit: React.MouseEventHandler<HTMLButtonElement> =
+    useCallback(() => {
+      console.log('edit');
+    }, []);
+
+  const handleMore: React.MouseEventHandler<HTMLButtonElement> =
+    useCallback(() => {
+      console.log('more');
+    }, []);
+
+  const handleDone: React.MouseEventHandler<HTMLButtonElement> =
+    useCallback(() => {
+      console.log('done');
+    }, []);
+
+  return (
+    <div className="relative flex h-38.75 w-44.75">
+      <div className="small-card-container bg-taaks-white relative z-0 flex h-38.75 w-44.75">
+        <div className="z-10 p-3">
+          <p className="text-taaks-subtitle pt-1 text-[12px] font-bold">
+            {deadline}
+          </p>
+          <p className="max-w-37.85 font-noto-sans-jp line-clamp-3 pt-6 text-[15px] leading-5 font-[600]">
+            {title}
+          </p>
+        </div>
+        <button
+          onClick={handleMore}
+          className="absolute bottom-2.25 left-4 z-10"
+        >
+          <MoreHorizontal className="size-5" />
+        </button>
+        <button
+          onClick={handleEdit}
+          className="absolute right-4 bottom-2.25 z-10"
+        >
+          <Edit className="size-5" />
+        </button>
+      </div>
+
+      {/* 右上のボタン */}
+      <button
+        onClick={handleDone}
+        className="bg-taaks-white hover:bg-primary absolute top-0 right-0 z-10 flex h-11.25 w-13.25 items-center justify-center rounded-full"
```

Comment:
riveになるからあまりこだわる必要ないのかもだけど、hoverつけるならtext-colorも変更して欲しい

```suggestion
        className="bg-taaks-white hover:bg-primary hover:text-primary-foreground absolute top-0 right-0 z-10 flex h-11.25 w-13.25 items-center justify-center rounded-full"
```